.gitlab-ci.yml

```diff
@@ -34,3 +34,24 @@ test:unit:
     - nats
   variables:
     DEVICECONNECT_MONGO_URL: "mongodb://mongo"
+
+test:acceptance:
+  stage: test
+  except:
+    - /^saas-[a-zA-Z0-9.]+$/
+  tags:
+    - docker
+  image: docker:19.03.5-dind
+  before_script:
+    - apk add docker-compose make
+  script:
+    - make acceptance-tests
+  after_script:
+    - make acceptance-tests-logs
+    - make acceptance-tests-down
+  artifacts:
+    expire_in: 2w
+    paths:
+      - tests/acceptance.*
+      - tests/coverage-acceptance.txt
+    when: always
```

Comment:
We are missing a `publish` job similar to https://github.com/mendersoftware/mendertesting/blob/master/.gitlab-ci-check-docker-acceptance.yml#L101